TedTalks/Utils/Enumerations.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Enumerations.swift
+//  TedTalks
+//
+//  Created by Miguel Angel Askar Rodriguez on 9/08/21.
+//
+
+import Foundation
+
+enum TedTalkFilters : String {
+    case allFilters = "All filters"
+    case event = "Event"
+    case mainSpeaker = "Main speaker"
+    case title = "Title"
+    case name = "Name"
+    case description = "Description"
+    
+    func getFilter() -> String {
```

Comment:
Given this is only used to return the user visible filter name ton be used in the UI maybe we can rename to `getName` or `getTitle`

Reply:
Ok, renamed to getName